### PR TITLE
fix: improve performance by using async function

### PR DESCRIPTION
**Pull request by ClarkXia**
Improve performance by using async function, such as esbuild onLoad https://esbuild.github.io/plugins/#on-load

### CODE REVIEW

File: packages/plugin-rax-compat/src/index.ts [hunk 1]
```diff
@@ -256,7 +256,7 @@ const inlineStylePlugin = () => {
     name: 'esbuild-inline-style',
     setup: (build) => {
       build.onLoad({ filter: /\.(css|sass|scss|less)$/ }, async (args) => {
-        const cssContent = fs.readFileSync(args.path, 'utf8');
+        const cssContent = await fs.promises.readFile(args.path, 'utf8');
```
Comment (wssgcg1213) [inline packages/plugin-rax-compat/src/index.ts:259]: 用了 fse 的 API 就命名为 fse 吧, 不然怪怪的   

`import fse from 'fs-extra';`

Comment (ClarkXia) [reply packages/plugin-rax-compat/src/index.ts:259]: 这里单纯就是 fs 模块

File: packages/ice/src/service/ServerRunner.ts [hunk 2]
```diff
@@ -247,7 +247,7 @@ class ServerRunner extends Runner {
                   if (res) {
                     const { contents, loader } = res;
                     if (['json', 'text'].includes(loader)) {
-                      code = `__ice_exports__.default = ${contents || JSON.stringify(fse.readFileSync(formatedId, 'utf-8'))}`;
+                      code = `__ice_exports__.default = ${contents || JSON.stringify(await fse.promises.readFile(formatedId, 'utf-8'))}`;
```
Comment (luhc228) [inline packages/ice/src/service/ServerRunner.ts:250]: 用 fse.readFile

File: packages/ice/src/service/analyze.ts [hunk 3]
```diff
@@ -96,7 +96,7 @@ export async function analyzeImports(files: string[], options: Options) {
   const importSet = new Set<string>();
   async function analyzeFile(filePath: string) {
     analyzedSet.add(filePath);
-    let source = fs.readFileSync(filePath, 'utf-8');
+    let source = await fs.promises.readFile(filePath, 'utf-8');
```
Comment (luhc228) [inline packages/ice/src/service/analyze.ts:99]: fs.readFile

**Review by luhc228 — APPROVED** [review]

**Review by wssgcg1213 — APPROVED** [review]

**Pull request merged**